phpBB/includes/acp/acp_attachments.php

```diff
@@ -1228,7 +1228,7 @@ function main($id, $mode)
 						'FILETIME'			=> $user->format_date((int) $row['filetime']),
 						'REAL_FILENAME'		=> (!$row['in_message']) ? utf8_basename((string) $row['real_filename']) : '',
 						'PHYSICAL_FILENAME'	=> utf8_basename((string) $row['physical_filename']),
-						'EXT_GROUP_NAME'	=> (!empty($extensions[$row['extension']]['group_name'])) ? $user->lang['EXT_GROUP_' . $extensions[$row['extension']]['group_name']] : '',
+						'EXT_GROUP_NAME'   	=> (!empty($user->lang[$extensions[$row['extension']]['group_name']])) ? $user->lang['EXT_GROUP_' . $extensions[$row['extension']]['group_name']] : $extensions[$row['extension']]['group_name'],
```

Comment:
I think what @Nicofuma means is `$user->lang($extensions[$row['extension']]['group_name'])` will produce the same result, as what your modification does.